From 6cccda2183cb1b0e6048ee57783f6db3c38fea8c Mon Sep 17 00:00:00 2001
From: Valentin Caron <valentin.caron@foss.st.com>
Date: Thu, 5 Dec 2024 16:03:47 +0100
Subject: [PATCH 1403/1695] dts: stm32: add st,proc-id to ipcc nodes in
 stm32mp251.dtsi

Add this property in ippc1 (0: CA35 is CPU1) and ipcc2 (1: CA35 is CPU2)
nodes.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/optee_os.git feea37eb8be010c4a3b3a162f14090109d2efb39 dts: stm32: add st,proc-id to ipcc nodes in stm32mp251.dtsi]
Signed-off-by: Valentin Caron <valentin.caron@foss.st.com>
Change-Id: I372d78c1d4bcbead04729f5f991caa890ba232c4
Reviewed-on: https://gerrit.st.com/c/mpu/oe/optee/optee_os/+/423179
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Reviewed-by: Arnaud POULIQUEN <arnaud.pouliquen@st.com>
Domain-Review: Thomas BOURGOIN <thomas.bourgoin@foss.st.com>
Reviewed-by: Michel JAOUEN <michel.jaouen@st.com>
Reviewed-by: Gwenael TREUVEUR <gwenael.treuveur@st.com>
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
Reviewed-by: Thomas BOURGOIN <thomas.bourgoin@foss.st.com>
---
 core/arch/arm/dts/stm32mp251.dtsi | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/core/arch/arm/dts/stm32mp251.dtsi b/core/arch/arm/dts/stm32mp251.dtsi
index a0312b372..41a9cf0fa 100644
--- a/core/arch/arm/dts/stm32mp251.dtsi
+++ b/core/arch/arm/dts/stm32mp251.dtsi
@@ -2250,6 +2250,7 @@
 			interrupts = <GIC_SPI 173 IRQ_TYPE_LEVEL_HIGH>;
 			interrupt-names = "rx";
 			clocks = <&rcc CK_BUS_IPCC1>;
+			st,proc-id = <0>;
 		};
 
 		ommanager: ommanager@40500000 {
@@ -3391,6 +3392,7 @@
 			compatible = "st,stm32mp25-ipcc";
 			reg = <0x46250000 0x400>;
 			clocks = <&rcc CK_BUS_IPCC2>;
+			st,proc-id = <1>;
 		};
 
 		stgenc: stgen@48080000 {
-- 
2.47.2

